test/unit/python/test_write_init_files.py

```diff
@@ -238,10 +238,10 @@ def test_simple_reg_constituent_write_init(self):
                       check_init_out, phys_input_out])
 
         # Generate registry files:
-        _, _, ic_names, constituents = gen_registry(filename, 'se', _TMP_DIR, 3,
-                                      _SRC_MOD_DIR, _CAM_ROOT,
-                                      loglevel=logging.ERROR,
-                                      error_on_no_validate=True)
+        _, _, ic_names, constituents, _ = gen_registry(filename, 'se', _TMP_DIR, 3,
```

Comment:
I think it would be good to add a new test (mirroring this one `test_simple_reg_constituent_write_init`, perhaps) that tests write_init_files.py when there is an vars_init_value list (to make sure the calls are added correctly)